### DISCUSSION TITLE
Automod и его настройка

### DISCUSSION THREAD

**Discussion by Travakyr** [stoatchat/discussions#803] [Q&A]
Доброй ночи, не могу настроить бота Automod при переходе по ссылке [Open Server Settings] выдаёт http://localhost:8080...... страница не найдена,как быть помогите пожалуйста

**Comment by DeclanChidlow** [comment]
Hi! I'm the developer of AutoMod. I'll jump on this ASAP. Thanks for the report.

**Reply by Travakyr** [reply]
Благодарю)
